infinoted/infinoted-options.c

```diff
@@ -1003,6 +1013,7 @@ infinoted_options_free(InfinotedOptions* options)
   g_free(options->certificate_file);
   g_free(options->certificate_chain_file);
   g_free(options->root_directory);
+  inf_ip_address_free(options->listen_address);
```

Comment:
According to malloc(3) ```free(NULL)``` does nothing, therefore the check is not strictly required as far as I understand.